docs/src/components/Navigation.vue

```diff
@@ -25,8 +27,18 @@ if (typeof window !== "undefined") {
   clearTimeout(resizeTimer)
   window.addEventListener("resize", () => {
     resizeTimer = setTimeout(applySizing, 200)
+
+    this.$once('hook:beforeUnmount', () => {
```

Comment:
`$once` is removed in Vue 3. 

`:hook` event prefix has also been replaced with `vnode-`

https://v3-migration.vuejs.org/breaking-changes/

Reply:
Apologies for the delayed response. Somehow I missed this notification for this. Updated now :)